client/components/list-end/index.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+export default function ListEnd() {
+	return <div className="list-end" />;
```

Comment:
I'd recommend hoisting the element, which avoids unnecessary calls to `createElement` and allows React to optimize diff-ing, an effect similar to what's automated with `babel-plugin-transform-react-constant-elements`:

https://babeljs.io/docs/plugins/transform-react-constant-elements/

```jsx
const listEnd = <div className="list-end" />;
export default function ListEnd() {
	return listEnd;
}
```